src/css/_BRTemplate.scss

```diff
@@ -5,3 +5,40 @@
 #item-navigator-modal {
   display: none
 }
+
+.BRpagecontainer {
+  --iconWidth: 16px;
+  --iconHeight: 24px;
+  --iconFillColor: #000;
+}
+
+/* BOOKMARKS - on page */
+.BRpagecontainer .bookmark-button {
```

Comment:
Toss this into a separate SCSS file to keep things tidy, and then import; similar to the text selection css file.